src/pkg/packaging/dir.proj

```diff
@@ -169,13 +169,10 @@
   </PropertyGroup>
 
   <Target Name="GenerateNugetPackages" DependsOnTargets="InitPackage" Condition="'$(UsePrebuiltPortableBinariesForInstallers)' == 'false'">
-    <ItemGroup>
-      <!-- The list of packages we are servicing -->
-      <PackageProjects Include="$(ProjectDir)src\managed\Microsoft.Extensions.DependencyModel\Microsoft.Extensions.DependencyModel.csproj" />
-    </ItemGroup>
 
     <ItemGroup Condition="'$(BuildAllPackages)' == 'true'">
```

Comment:
Let's hope we don't need any more fixes in these for 2.0.x.

One that is on my radar is needing to change the RID generation code for RHEL6 since `/etc/os-release` doesn't exist.  Is RHEL6 officially supported in 2.0.x?

@janvorli